mmv1/products/redis/Cluster.yaml

```diff
@@ -250,6 +250,56 @@ properties:
     description: |
       System assigned, unique identifier for the cluster.
     output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    description: "The automated backup config for a instance."
+    properties:
+      - name: 'automatedBackupMode'
+        type: Enum
+        description: |
+          "Optional. The automated backup mode. If the mode is disabled, the other fields will be ignored."
+          1. `AUTOMATED_BACKUP_MODE_UNSPECIFIED`: Default value. Automated backup config is not specified.
+          2. `DISABLED`: Automated backup config disabled.
+          3. `ENABLED`: Automated backup config enabled.
+        enum_values:
+          - 'AUTOMATED_BACKUP_MODE_UNSPECIFIED'
```

Comment:
We can remove this from here + the description and just make `DISABLED` the default.